core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/InProcessEmbeddingProcessor.java

```diff
@@ -120,20 +124,31 @@ InProcessEmbeddingBuildItem e5_small_v2() {
     @Record(ExecutionTime.RUNTIME_INIT)
     void exposeInProcessEmbeddingBeans(InProcessEmbeddingRecorder recorder,
             List<InProcessEmbeddingBuildItem> embeddings,
-            BuildProducer<SyntheticBeanBuildItem> beanProducer) {
```

Comment:
You can't remove this, it's necessary to keep the `beanProducer.produce(...)` call, otherwise the bean will not exist

Reply:
Ok,I thought you said I don't have to produce it. I will fix it. Weird that the other integration tests are working ?